schema.sql

```diff
@@ -0,0 +1,105 @@
+DROP DATABASE readme;
+
+CREATE DATABASE IF NOT EXISTS readme
+  DEFAULT CHARACTER SET utf8
+  DEFAULT COLLATE utf8_general_ci;
+
+USE readme;
+
+CREATE TABLE users (
+  id INT AUTO_INCREMENT PRIMARY KEY,
+  email VARCHAR(128) NOT NULL UNIQUE,
+  login VARCHAR(255) NOT NULL UNIQUE,
+  password CHAR(64) NOT NULL,
+  avatar VARCHAR(255) DEFAULT NULL,
+
+  UNIQUE INDEX email_index (email),
```

Comment:
В имени индекса желательно добавлять имя таблицы, а суфикс `_index` не имеет смысла, т.к. это имя индекса

```suggestion
  UNIQUE INDEX users_email (email),
```